src/components/LanguageButtons.tsx

```diff
@@ -18,29 +18,53 @@ const Container = styled.div<FlexboxProps & LayoutProps & SpaceProps>`
   ${space};
 `;
 
-const LangButton = styled.button<SpaceProps & TypographyProps>`
+const LangButton = styled.button<
+  SpaceProps & TypographyProps & { isActive: boolean }
+>`
   background: transparent;
   border: none;
   ${typography};
   ${space};
   transition: transform 0.2s;
+  ${(props) => props.isActive && ActiveLang}
   &:hover {
     transform: scale(1.05);
+  
+`;
+
+const ActiveLang = css`
+  text-decoration-line: underline;
 `;
 
 const LanguageButtons = () => {
   const fontSizes = [1, 2, 3, 4];
+
+  const [activeLang, setActiveLang] = useState(i18next.language);
```

Comment:
hmm the `i18next.language` doesnt seem to work like this since `i18next` isnt initialised by the time this component is rendered, i think remove the initial state as its not vital and then we can get this in

Reply:
once you remove `i18next.language` it will give you a typescript error:

you need to specifically set the data type of `activeLang` (since it can no longer be inferred by the `i18next.language`)

Reply:
i'm not sure I totally understand when you say it doesn't work - I get that you're referring to this:

> NB. for i18next.language i looked at their documentation i didnt know it, essentially on the first load when the initial state is set you need to know the active language

I can't see why we would take out the `i18next.language` because how then are we able to access the language values in the `118next` object?

Reply:
ah okay, I understand what you're saying I think. Is this to do with code splitting and only loading the translations of the active language rather than all translations at once?

Reply:
then again perhaps I'm still missing the point because as far as I understand from the documentation a Suspense comes part and parcel of the `useTranslation` hook so I can't see why the visual loading sequence would be the issue here. I think I might need a bit of a prompt :)

Reply:
The language buttons dont have an underline when you load the page, but when you click a language button it does have an underline. This is the minor bug im talking about. (there is a problem with the initial state of the `activeLang` piece of state

and im saying lets not fix it now

Reply:
Oh I geddddit. Actually, that's not something I'd even contemplated - UX wise it didn't feel weird to me! But now you mention it I can't unsee it lol. But yeah happy to fix later down the line.